test-base.cfg

```diff
@@ -208,19 +208,26 @@ commands =
 
 # bin/pycodestyle : pycodestyle validation of the packages source.
 [pycodestyle]
-recipe = zc.recipe.egg
+recipe = zc.recipe.egg:scripts
 eggs = pycodestyle
-args = ['--exclude=tests', '--config=${pycodestyle-cfg:cfgfile}']
```

Comment:
Did you remove `--exclude=tests` by accident or on purpose? It's not mentioned in the PR body.